doc-Self_Service_User_Interface_Guide/topics/Navigating_Self_Service_UI.adoc

```diff
@@ -28,7 +28,12 @@ Clicking the menu icon (image:ssui-administration-ui-icon.png[Administration UI
 
 Help::
 
-Click the help icon to access the options to open *Documentation*, *Red Hat Customer Portal*, and *About*. Click *About* to see information about your Self Service environment, such as the logged in user name and role, the browser details, and more importantly, the version number of your Self Service user interface, which can be useful for support purposes.
+Click the help icon to access the options to open *Documentation*, *Red Hat Customer Portal*, and *About*. Click *About* to see information about your Self Service environment, such as the logged in user name and role, the browser details, and importantly, the Self Service user interface version number, which can be useful for support purposes.
+
+[NOTE]
```

Comment:
Hey Suyog. I would recommend breaking this NOTE into two sentences. System administrators can change....

Reply:
Updated, thanks.